packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -213,8 +213,8 @@ function isSideloadingSupportedForWebHost(app: OfficeApp): boolean {
   return false;
 }
 
-function hasOfficeVersion(targetVersion: string, currentVersion: string): boolean {
-  return semver.gte(currentVersion, targetVersion)
+function hasOfficeVersion(targetVersion: string, currentVersion: string | undefined): boolean {
```

Comment:
This function should only take string, and not take undefined.

You should put the logic about the fact that the getOutlookVersion() function can return undefined outside of here.